dll/win32/comctl32/lang/comctl_Zh.rc

```diff
@@ -98,3 +98,83 @@ BEGIN
   LTEXT         "当前工具栏按钮(&T):", -1,194,5,120,10
   LISTBOX       IDC_TOOLBARBTN_LBOX, 194,17,120,100,LBS_NOTIFY | LBS_OWNERDRAWFIXED | LBS_HASSTRINGS | LBS_NOINTEGRALHEIGHT | LBS_DISABLENOSCROLL | WS_BORDER | WS_VSCROLL | WS_HSCROLL | WS_TABSTOP
 END
+
+LANGUAGE LANG_CHINESE, SUBLANG_CHINESE_TRADITIONAL
```

Comment:
Ok I verified with Wine, even them have two separate files for zh-CN and zh-TW accordingly:

- https://source.winehq.org/git/wine.git/blob/f69d4a865f926aa5c4a9c55cfe4d2dbc10746e5c:/po/zh_CN.po#l408
- https://source.winehq.org/git/wine.git/blob/f69d4a865f926aa5c4a9c55cfe4d2dbc10746e5c:/po/zh_TW.po#l403